portal/static/portal/sass/styles.scss

```diff
@@ -811,74 +818,122 @@ form {
 }
 
 .arrows {
-  @include _padding(20px, 0px, 0px, 0px);
+  @include _padding(20px, 40px, 0px, 5px);
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: center;
+  text-align: center;
 }
 
-.progress-arrow {
-  @include _margin(0px, -10px, 0px, 0px);
-  display: inline-block;
+.onboarding__step {
+  @include _margin(0px, 8px, 0px, 8px);
+  align-items: center;
+  display: inline-flex;
+  height: 50px;
+  justify-content: center;
+  position: relative;
+  text-align: center;
+  width: 250px;
 
   img {
-    max-width: none;
-  }
-
-  * {
-    display: inline-block;
+    @include _margin(0px, 10px, 5px, 25px);
   }
 
 }
 
-.progress-arrow__back {
-  @include _margin(0px, -3px, 0px, 0px);
+.onboarding__step__text {
+  align-items: center;
+  display: inline-flex;
+  justify-items: center;
 }
 
-.progress-arrow__body {
-  @include _margin(0px, -3px, 0px, 0px);
-  @include _padding(5px, 10px, 17px, 15px);
-  height: 50px;
-  text-align: left;
-  width: 215px;
-
-  * {
-    display: inline-block;
-  }
+.onboarding__step--done {
+  background: $color-onboarding-step-done-background;
+  border-bottom: 1px solid $color-background-secondary;
+  border-top: 1px solid $color-background-secondary;
+  color: $color-onboarding-step-done-text;
+}
 
-  p {
-    @include _padding(0px, 0px, 0px, 10px);
-  }
+.onboarding__step--done:before {
+  background: transparent;
+  border-right: 1px solid $color-background-secondary;
+  border-top: 1px solid $color-background-secondary;
+  content: "";
+  height: 35px;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transform: rotate(45deg);
+  transform-origin: 0 0;
+  width: 35px;
+}
 
+.onboarding__step--done:after {
+  background: transparent;
+  border-right: 1px solid $color-background-secondary;
+  border-top: 1px solid $color-background-secondary;
+  content: "";
+  height: 35px;
+  position: absolute;
+  right: -35px;
+  top: 0;
+  transform-origin: 0 0;
+  transform: rotate(45deg);
+  width: 35px;
 }
 
-.progress-arrow__body--undone {
-  background-color: $color-background-tertiary;
-  border-bottom: 1px solid $color-background-tertiary;
-  border-top: 1px solid $color-background-tertiary;
+.onboarding__step--in-progress {
+  background: $color-onboarding-step-in-progress-background;
 
   p {
-    color: $color-text-secondary;
+    color: $color-onboarding-step-in-progress-text;
   }
 
 }
 
-.progress-arrow__body--in-progress {
-  background-color: $color-background-secondary;
-  border-bottom: 1px solid $color-background-secondary;
-  border-top: 1px solid $color-background-secondary;
+.onboarding__step--in-progress:before {
+  border-bottom: 25px solid transparent;
```

Comment:
You set 25px over and over again.  Use a variable.

Also, if it's related to that 35px up there ^^, make them explicitly related using Sass calculations. eg.
`$border: $position - 10px`